src/conn/conn_dhandle.c

```diff
@@ -465,7 +509,7 @@ __wt_conn_dhandle_open(WT_SESSION_IMPL *session, const char *cfg[], uint32_t fla
         WT_ERR(__wt_btree_open(session, cfg));
         break;
     case WT_DHANDLE_TYPE_TABLE:
-        WT_ERR(__wt_schema_open_table(session, cfg));
```

Comment:
The `cfg` parameter was not used, and caused me confusion. I also opened WT-6878 to go a step further and remove `cfg` from the `\_\_wt_btree_open` call above.